packages/minifiers-css/urlrewriting-tests.js

```diff
@@ -32,12 +32,12 @@ Tinytest.add("minifiers - url rewriting when merging", function (test) {
   t('http://', 'http://', 'malformed URL');
 
   parseOptions.source = 'application/client/dir/other-style.css';
-  t('./image.png', '/image.png', 'base path is root');
-  t('./child/image.png', '/child/image.png', 'child directory from root');
-  t('child/image.png', '/child/image.png', 'child directory from root');
-  t('/image.png', '/image.png', 'absolute url');
-  t('"/image.png"', '"/image.png"', 'double quoted url');
-  t("'/image.png'", "'/image.png'", 'single quoted url');
+  t('./image.png', 'image.png', 'base path is root');
```

Comment:
Sure, it’s actually pretty simple. Let’s launch a Meteor app with:

    ROOT_URL="http://localhost:3000/subpath" meteor

and let's assume that the served CSS file contains something like

```css
background: url('/image.png');
```

The browser will resolve this resource by GETing the file `http://localhost:3000/image.png` whereas the file is actually served at `http://localhost:3000/subpath/image.png` resulting in a Not Found error. The problem is that absolute paths are relative to the domain name root, not necessarily the application root (in case the app is served under a sub-path).

I will provide a reproduction to make the issue even clearer.